.github/workflows/run-tests.yml

```diff
@@ -13,13 +13,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest]
-        java-version: [8, 17]
-        include:
-          - os: macos-latest
-            java-version: 17
-          - os: macos-latest
-            java-version: 11
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        java-version: [11, 17]
```

Comment:
Why is that? Is this just convenient or necessary. I'd like to avoid dropping support for a Java version in a minor release. If we need to drop it, we should bump to `0.11.0`. Keeping bin-compat with `0.10.x` is a bonus feature.

Reply:
Just convenience really, but i realized we already have `os.Internal.transfer` so I can just use that just as conveniently

Reply:
Seems like there's some issue with zip filesystem on windows java 8, so need to bump that to java 11 as well

Reply:
I think given that we now have 2/3 OSs only testing on Java 11, let's just bump the official supported version to 11. People can continue to try using it on 8, it'll just be at their own risk